### DISCUSSION TITLE
Take advantage of the GitHub NPM registry

### DISCUSSION THREAD

**Discussion by PseudoNinja** [Modernizr/Modernizr#2661] [Ideas]
Any chance we can see some packages released to the GitHub NPM registry to make consumption a bit easier. 

A couple hours of work could go a long way to help the community.

**Comment by rejas** [comment]
Hi @PseudoNinja not sure what you mean by the github npm registry. We publish our packages on npm already, doesnt github mirror that?

**Reply by roblarsen** [reply]
https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages

It's super easy

**Reply by rejas** [reply]
Since I am a little distracted ATM I got to ask: Is this something that can be done via a PR or does someone have to change settings in the repo here @roblarsen ?

**Reply by roblarsen** [reply]
You can do it with a PR, IIRC. You need a secret from the repo, but GitHub generates it automatically. It's not like the secrets you need with npm. If it's possible without a settings change, I can create a PR.

**Reply by Markel** [reply]
Sounds okay to me, the more options for developers to download the packages the better (also taking space twice on Microsoft servers brings me some satisfaction 😆😅). If you can change it with an easy PR I would, personally, say that you go for it.
